plugin/editor/blocks/text-block/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/elements';
+import { registerBlock, Editable } from '@wordpress/blocks';
+
+registerBlock( 'wp', 'Text', {
+	edit( state, onChange ) {
+		return createElement( Editable, {
+			value: state.value,
+			onChange: ( value ) => onChange( { value } )
+		} );
+	},
+	save( state ) {
+		return createElement( 'p', null, state.value );
```

Comment:
>Why do we need it to be an element (which is HTML + events instead of just HTML)?

It's important to point out that this is as simple as blocks get. For everything else, `createElement`'s purpose doubles as a convenient DOM builder.

While also true that an element could be stringified without too much trouble, (a) it's still nicer if you don't have to, and (b) I find the consistency with `edit` leads to a pleasant duality.